src/tests/multihost/alltests/test_automount_from_bash.py

```diff
@@ -0,0 +1,993 @@
+        user_info = {'objectClass': ['nisObject'.encode('utf-8'),
+                                     'top'.encode('utf-8')],
+                     'nisMapName': 'auto.direct'.encode('utf-8'),
+                     'cn': '/largepathfolder1/largepathfolder2/largepathfolder3/'
+                           'largepathfolder4/largepathfolder5/largepathfolder6/largepathfolder7/'
+                           'largepathfolder8/largepathfolder9/largepathfolder10/largepathfolder11/'
+                           'largepathfolder12/largepathfolder13/largepathfolders'.encode('utf-8'),
+                     'nisMapEntry': f'-fstype=nfs,rw {master_server}:/export/largepathfolder'.encode('utf-8')}
+        ldap_inst.add_entry(user_info, user_dn)
+        client.run_command("service autofs restart")
+        tools.clear_sssd_cache()
+        client.run_command("cd /largepathfolder1/largepathfolder2/largepathfolder3/largepathfolder4/"
+                           "largepathfolder5/largepathfolder6/largepathfolder7/largepathfolder8/"
+                           "largepathfolder9/largepathfolder10/largepathfolder11/largepathfolder12/"
+                           "largepathfolder13/largepathfolders;cd -")
+        assert f"{master_server}:/export/largepathfolder" in client.run_command("mount").stdout_text
+
+    @staticmethod
+    def test_autofs_dumpmaps(multihost, backupsssdconf, ldap_autofs):
+        """
+        :title: SSSD frequently fails to return automount maps from LDAP bz967636
+        :id: 499275c6-9897-11ed-8e99-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=967636
+        :steps:
+            1. Have lots of automount entries in an LDAP server (~250 in our case).
+            2. On an SSSD+autofs+LDAP client, run `automount --dumpmaps` 10 times.
+            3. Behold: you will only get automount maps from LDAP ~30% of the time.
+        :expectedresults:
+            1. 250 automount entries should be created
+            2. Sssd should not crash
+            3. automount --dumpmaps should always return a
+                complete set of automount maps from  LDAP.
+        """
+        # SSSD frequently fails to return automount maps from LDAP bz967636
+        client = multihost.client[0]
+        master_server = multihost.master[0].sys_hostname
+        ldap_uri = f'ldap://{multihost.master[0].sys_hostname}'
+        ldap_inst = LdapOperations(ldap_uri, ds_rootdn, ds_rootpw)
+        # sssd automount maps from LDAP test BZ 967636
+        client.run_command("service autofs restart")
+        user_dn = f'nisMapName=auto.share,ou=mount,{ds_suffix}'
+        user_info = {'objectClass': ['nisMap'.encode('utf-8'),
+                                     'top'.encode('utf-8')],
+                     'nisMapName': 'auto.share'.encode('utf-8')}
+        ldap_inst.add_entry(user_info, user_dn)
+        # Test autofs dumpmaps consistency.
+        for i in range(250):
```

Comment:
Could it be moved to conftest.py or is it better here?

Reply:
its used in only one test suit , not reusable